tests/test_unit.py

```diff
@@ -6623,6 +6711,20 @@ def testWarnUsingLshiftForward(self):
             except Exception as e:
                 self.fail("raised warning when it should not have")
 
+    def testAssertParseAndCheckDict(self):
+        import pyparsing as pp
```

Comment:
I'm pretty sure there is a module-level `import pyparsing as pp`, so shouldn't need to repeat here. For that matter, should not need to import `Word`, `alphas`, etc. anywhere, just reference as `pp.Word`, `pp.alphas`, etc.

Reply:
You're right, there is. I was shamelessly stealing from other tests that have unnecessary imports (possibly written before the module level import.) I've made a note to go through and clean those up throughout the file.